packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { Suspense, useEffect, useState } from 'react';
+
+import useIsBrowser from './useIsBrowser';
+import useIsIntersecting from './useIsIntersecting';
+
+import type { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+const Overlay = () => (
```

Comment:
This overlay darkens the blurhash placeholder in the same way that the eventually-loading player would